gadgets/src/range.rs

```diff
@@ -0,0 +1,153 @@
+//! Specialized version to replace LtChip for u32/u64/u128 check
```

Comment:
The chip is not really related to LtChip.

Reply:
This chip is created for a drop-in replace for original approach using LtChip to check [0, uXX::MAX].